lang/cs/build.props

```diff
@@ -94,10 +97,11 @@ under the License.
       <Using Namespace="System.Xml.Linq"/>
       <Code Type="Fragment" Language="cs">
         <![CDATA[
-Version = XDocument.Load(Path.Combine(ProjectFolder, "pom.xml")).Descendants()
+          Version = XDocument.Load(Path.Combine(ProjectFolder, "pom.xml")).Descendants()
             .Where(x => x.Name.ToString().Contains("version"))
             .FirstOrDefault().Value;
-]]>
+          NugetVersion = Version.Replace("-incubating","") + "-" + SnapshotNumber;
```

Comment:
I tried it, got error "The special version part cannot exceed 20 characters" from NuGet.exe. Looks like our version number is too long id adding incubating.